rotkehlchen/chain/gnosis/modules/gnosis_pay/constants.py

```diff
@@ -0,0 +1,13 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+GNOSIS_PAY_CASHBACK_ADDRESS: Final = string_to_evm_address('0xCdF50be9061086e2eCfE6e4a1BF9164d43568EEC')  # noqa: E501
+
+CPT_GNOSIS_PAY: Final = 'gnosis pay'
+GNOSIS_PAY_CPT_DETAILS: Final = CounterpartyDetails(
+    identifier=CPT_GNOSIS_PAY,
+    label='Gnosis Pay',
+    image='gnosis.svg',
```

Comment:
gnosis pay logo is different. It's lighter green. Need to find it add it as a file to the PR and reference it here

![2024-09-12_23-47](https://github.com/user-attachments/assets/d93dde52-21de-4e5c-8eb8-ff703e5840ce)